.gitignore

```diff
@@ -0,0 +1,2 @@
+*.code-workspace
+Platform/ADL/ADL-P/Softstrap/Native/*.*
```

Comment:
Not sure if you want this in the .gitignore. We shouldn't run into this and it feels like the wrong type of information for this repo.